cli/cmd/runner.go

```diff
@@ -77,6 +77,7 @@ type runnerArgs struct {
 
 	customerLsIncludeTest bool
 
+	archiveCustomerNameOrId           string
```

Comment:
Nit, for consistency with existing `nounVerbObject`names:
```suggestion
	customerArchiveNameOrId           string
```

Reply:
Done!